whylog/tests/tests_config/__init__.py

```diff
@@ -7,6 +7,7 @@
 from whylog.config.parsers import RegexParserFactory
 from whylog.config.rule import RegexRuleFactory
 from whylog.teacher.user_intent import UserConstraintIntent, UserParserIntent, UserRuleIntent
+from whylog.tests.utils import path_test_files
 
 # Constraint types
 identical_constr = "identical"
```

Comment:
This is not what my PR do. I only move path_test_files to utils.py file becouse I need it in different place. Ask @kgromadzki to change it.